server/src/main/java/org/elasticsearch/index/query/GeoShapeQueryBuilder.java

```diff
@@ -166,6 +170,11 @@ public GeoShapeQueryBuilder(StreamInput in) throws IOException {
             indexedShapeType = in.readOptionalString();
             indexedShapeIndex = in.readOptionalString();
             indexedShapePath = in.readOptionalString();
+            if (in.getVersion().onOrAfter(Version.V_7_0_0_alpha1)) {
```

Comment:
This should be 6.4.0, right?

Reply:
Yes, eventually, after/during backport. If I make it 6.4.0 now, it will never pass BWC tests.

Reply:
Cool. I figure the branch overrides like `-Dtests.bwc.refspec.6.x=` would be good for that sort of thing.